pkg/client/dtclient/settings_client.go

```diff
@@ -202,7 +202,7 @@ func (d *DynatraceClient) upsertSettings(ctx context.Context, obj SettingsObject
 			props = append(props, fmt.Sprintf("(%v = %v)", k, v))
 		}
 
-		log.WithCtxFields(ctx).Info("Updating existing object %q with matching unique properties: %v", matchingObject.object.ObjectId, strings.Join(props, ", "))
+		log.WithCtxFields(ctx).Debug("Updating existing object %q with matching unique properties: %v", matchingObject.object.ObjectId, strings.Join(props, ", "))
```

Comment:
@Laubi didn't we discuss not just setting this to debug completely last week? :/ 
As said, I view this as the same type of INFO the logs on creating or updating configs by name or ID.
This tells me that an existing object was updated, rather than a new setting being created.

I agree to moving the full list of matching properties to debug, but as said last week, IMO this shouldn't fully vanish into Debug logs.

FYI @warber (not to worry for the bugfix release, but we should re-discuss this handling)

Reply:
> This tells me that an existing object was updated, rather than a new setting being created.

@UnseenWizzard IMO the way to go is to introduce a field of the general response struct the returned by the (new) api clients that can be used to check whether calling e.g. `Upsert` resulted in a modification or a creation. Based on that the deployStatus field can be set accordingly.